text/0070-document-sensitive-data-collected.md

```diff
@@ -0,0 +1,87 @@
+
+This RFC is related to [RFC-0062 Controlling PII and Credentials in SDKs](https://github.com/getsentry/rfcs/pull/62).
+
+# Motivation
+
+We collect a lot of data, and transparency creates trust. This documentation will make it easier for customers to choose Sentry because they know that their users data is in good hands.
+It will also make it easier for our customers to be GDPR compliant. Companies that deal with data related to persons in the european union need to create a record of what data they are processing.
+This documentation will make our customers lifes way easier while creating these records.
+This will probably be a big selling point for larger customers.
+
+# Background
+
+After a data incident and a meeting with legal, we said that we need to take data issues to the next level.
+
+# Options Considered
+
+## A) Table in docs of each integration
+
+Have a hand written (and maintained) table in the description that shows people in an easy to grasp way what data is collected. It also shows how the data collection is changed when certain options (like `sendDefaultPII`) are changed.
+
+Here a example on how this could look like. (After talking with our designer Jesse, having two tables makes it way easier to ingest the information)
+
+For issues:
+https://sentry-docs-git-antonpirker-python-fastapi-sensitive-data.sentry.dev/platforms/python/guides/fastapi/#data-collected-by-issue-reporting
+
+And for performance:
+https://sentry-docs-git-antonpirker-python-fastapi-sensitive-data.sentry.dev/platforms/python/guides/fastapi/#data-collected-measuring-performance
+
+The elements in the table can be different for different kinds (frontend, backend, mobile) of SDKS.
+
+Here a list of all sensitive data that is collected:
+
+- HTTP Headers (`event.request.headers`)
+- HTTP Cookies (`event.request.cookies`)
+- HTTP Request Body (`event.request.data`)
+- Log Entry Params (`event.logentry.params`)
+- Logged in User (`event.user`)
+- Users IP address (`event.user`)
+- Breadcrumb Values (`event.breadcrumbs.values -> value.data`)
+- Local vars in Exceptions (`event.exception.values -> value.stacktrace.frames -> frame.vars`)
+- Span Data (`event.spans -> span.data`)
+- ... more to be defined ...
+
+Pros:
+
+- Easy understandable and nice to read documentation
+
+Cons:
+
+- Documentation need to be kept up to date with seperate PR in `sentry-docs` repo when changes to SDK are made
+- Documentation for different versions of the SDK not solved yet
+
+## B) Automatic documentation creation
+
+If we go with _Option B)_ in [RFC-0062 Controlling PII and Credentials in SDKs](https://github.com/getsentry/rfcs/pull/62) we could add doc strings in the code of the implemented `EventScrubber` and then generate documentation from this code to render a table similar to the one in Option A) in this RFC.
```

Comment:
This RFC was created because people were asking for documentation on what data is collected, so only having it in the code is not enough. :-) 

But I guess when we have the EventScrubber we can have a documentation on how the event scrubbing works in general and then point to the default `EventScrubber`s code to the people that want to know the details.

Reply:
Yeah I was referring to the other RFCs outcome of having a default filtering logic which could then also serve as docs (at least for people able to read the code).